app/src/main/java/org/p2p/wallet/user/interactor/UserInteractor.kt

```diff
@@ -100,6 +100,11 @@ class UserInteractor(
         mainLocalRepository.getUserTokens()
             .sortedWith(TokenComparator())
 
+    suspend fun getNonEmptyUserTokens(): List<Token.Active> =
```

Comment:
getNonZero maybe?
how token can be empty?

Reply:
yeah, okay, will rename it